api/src/Controller/Stores/PutProduct.php

```diff
@@ -23,10 +23,6 @@ public function buildResource(Request $request, Response $response, $params): ar
 
         $product = $this->getProduct($params, $request, $response, $error);
```

Comment:
SonarCloud rightly bitches about this being a useless assignment. It actually occurs to me that we can use the `rowCount` trick here and `PutStore`, too (if the update affects 0 rows it's a 404). So I'll be updating with that, maybe later tonight.